From ed5971df25fe17ccdfd4b760d76b9d1083e6fbbd Mon Sep 17 00:00:00 2001
From: Robert Chiras <robert.chiras@nxp.com>
Date: Mon, 27 Nov 2017 17:14:26 +0200
Subject: [PATCH 2982/5242] MLK-16926-4: arm64: dts: fsl-imx8mq-evk: Add sync
 polarity for LCDIF use-cases

commit  493301a2b3645a6de89555e74c6249687484cbb3 from
https://source.codeaurora.org/external/imx/linux-imx.git

For some reasons, the sync polarity of the eLCDIF when used with NWL DSI
controller needs to be HIGH, so set it in the DTS nodes.

Signed-off-by: Robert Chiras <robert.chiras@nxp.com>
Reviewed-by: Laurentiu Palcu <laurentiu.palcu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../dts/freescale/fsl-imx8mq-evk-lcdif-adv7535.dts |    1 +
 .../dts/freescale/fsl-imx8mq-evk-lcdif-rm67191.dts |    1 +
 2 files changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-lcdif-adv7535.dts b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-lcdif-adv7535.dts
index 47398833..133ace3c 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-lcdif-adv7535.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-lcdif-adv7535.dts
@@ -72,6 +72,7 @@
 &mipi_dsi {
 	status = "okay";
 	as_bridge;
+	sync-pol = <1>;
 
 	port@1 {
 		mipi_dsi_in: endpoint {
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-lcdif-rm67191.dts b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-lcdif-rm67191.dts
index 72c7530..884cdad 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-lcdif-rm67191.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-lcdif-rm67191.dts
@@ -62,6 +62,7 @@
 &mipi_dsi {
 	status = "okay";
 	as_bridge;
+	sync-pol = <1>;
 	assigned-clocks = <&clk IMX8MQ_CLK_DSI_PHY_REF_SRC>,
 			  <&clk IMX8MQ_CLK_DSI_CORE_SRC>,
 			  <&clk IMX8MQ_VIDEO_PLL1_REF_SEL>,
-- 
1.7.9.5

